docs/versions/0.0.0.md

```diff
@@ -234,12 +234,9 @@ Requirements
 3. The document element MUST contain zero or more `file` elements, and MUST NOT contain other elements.
    <!-- Skip if XSD is used -->
 
-4. Each `file` element MUST have a `filepath` attribute which contains the bag local path to the payload file described. 
-   When an `original-filepaths.txt` exists, rule 2.7.2 applies.
-   Directories and non-payload files MUST NOT be described by a `file` element. 
+4. Each `file` element MUST have a `filepath` attribute which records the correspondence with a payload file.
 
-5. There MUST NOT be more than one `file` element corresponding to a payload file and every payload file MUST be described
-   by a `file` element.
+5. (a) The `filepath` attribute MUST contain the bag relative path of the payload file, (b) OR if `original-filepaths.txt` exists, rule 2.7.2 applies.
```

Comment:
it doesn't say anymore that every payload file must be represented in the files.xml

```suggestion
5. (a) The `filepath` attribute MUST contain the bag relative path of the payload file and every payload file MUST be represented exactly once in the files.xml, (b) OR if `original-filepaths.txt` exists, rule 2.7.2 applies.
```